app/assets/stylesheets/nav.scss

```diff
@@ -3,13 +3,13 @@ $font-sans: Helvetica, Arial, sans-serif;
 $button-color: #1E90FF;
 
 #magellan_bar {
-  min-width:0;
-     width: auto;
-    margin:0 auto;
-    margin-left: 5%;
-    list-style-type: none;
-
+  min-width: 0;
```

Comment:
Properties should be ordered list-style-type, margin, margin-left, min-width, width